x/incentive/types/params.go

```diff
@@ -75,8 +75,11 @@ func (p Params) Validate() error {
 			return fmt.Errorf("cannot have duplicate reward denoms: %s", reward.Denom)
 		}
 		rewardDenoms[reward.Denom] = true
+		if !reward.AvailableRewards.IsValid() {
+			return fmt.Errorf("invalid reward coins %s for %s", reward.AvailableRewards, reward.Denom)
+		}
 		if !reward.AvailableRewards.IsPositive() {
```

Comment:
not required since IsValid already checks for all positive coins

Reply:
Test with amount == 0 was failing. I believe IsValid is only checking non-negative. Has this changed in v0.38.3?